packages/wrangler/src/dev/local.tsx

```diff
@@ -497,6 +497,14 @@ export function setupMiniflareOptions({
 	miniflareCLIPath: string;
 	forkOptions: string[];
 } {
+	const defaultRules = DEFAULT_MODULE_RULES.slice();
```

Comment:
```suggestion
	const defaultRules = DEFAULT_MODULE_RULES;
```

Reply:
ah, I see, yeah ok. I'd prefer we not have to do this but ok

Reply:
It was just a safety measure, I can remove if desired 🙂